src/main/java/org/skriptlang/skript/bukkit/itemcomponents/equippable/EquippableWrapper.java

```diff
@@ -0,0 +1,49 @@
+package org.skriptlang.skript.bukkit.itemcomponents.equippable;
+
+import ch.njol.skript.bukkitutil.ComponentWrapper;
+import ch.njol.skript.util.ItemSource;
+import org.bukkit.Material;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.bukkit.inventory.meta.components.EquippableComponent;
+import org.skriptlang.skript.lang.converter.Converter;
+
+import java.util.function.BiConsumer;
+
+/**
+ * {@link ComponentWrapper} class for {@link EquippableComponent}.
```

Comment:
this isn't really saying anything `extends ComponentWrapper<EquippableComponent>` doesn't already say